src/components/SongHistory.js

```diff
@@ -0,0 +1,42 @@
+import React, {Component} from 'react';
+
+class SongHistory extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      displayList: false,
+    }
+  }
+
+  toggleDisplay = () => {
+    this.setState({
+      displayList: !this.state.displayList
+    })
+  }
+
+  render(){
+    const {songHistory} = this.props;
+    const songs = songHistory.map(song => song.song).reverse()
+    return(
+      <div id='recent-song-history' onClick={this.toggleDisplay}>
+        {
+          this.state.displayList &&
+          <div id='recent-song-list'>
+            {songs.map(song =>
+              <div key={song.id} class='recent-song-info'>
+                <img src={song.art} alt={`Album Title: ${song.album}`}/>
+                <p>
+                  <b>Song Title: {song.title}</b><br />
+                  Artist: {song.artist}

```

Comment:
Out of curiosity, why don't we just use 2 `<p>` elements, one for song name and one for artist?

I also agree with Ahmad on the removal of the text `Song Title:` and `Artist:`, and maybe increase the font size of song name to differentiate them, but it looks fine either way.